tools/tupctl.c

```diff
@@ -429,6 +439,99 @@ static int do_set_sensor_value(int argc, char *argv[])
     return ret;
 }
 
+static int do_get_input_value(int argc, char *argv[])
+{
+    TupMessage msg;
+    int effect_slot_id;
+    int input_id;
+    int ret;
+
+    if (argc != 2) {
+        printf("'get_input_value' arguments: <effect slot id> <input-id>\n");
+        return -EINVAL;
+    }
+
+    ret = sscanf(argv[0], "%d", &effect_slot_id);
+    ret += sscanf(argv[1], "%d", &input_id);
+    if (ret != 2) {
+        printf("failed to parse arguments\n");
+        return -EINVAL;
+    }
+    printf("Slot : %d\ninput : %d\n", effect_slot_id, input_id);
+
+    tup_message_init_get_input_value_simple(&msg, effect_slot_id, input_id);
+    ret = tup_context_send(&tup_ctx, &msg);
+    tup_message_clear(&msg);
+    return ret;
+}
+
+static int do_set_input_value(int argc, char *argv[])
+{
+    TupMessage msg;
+    int effect_slot_id;
+    int input_id;
+    int input_value;
+    int ret;
+
+    if (argc != 3) {
+        printf("'set_input_value' arguments: <effect slot id> <input-id> <value>\n");
+        return -EINVAL;
+    }
+
+
+    ret = sscanf(argv[0], "%d", &effect_slot_id);
+    ret += sscanf(argv[1], "%d", &input_id);
+    ret += sscanf(argv[2], "%d", &input_value);
+    if (ret != 3) {
+        printf("failed to parse arguments\n");
+        return -EINVAL;
+    }
+    printf("Slot : %d\ninput : %d\nvalue : %d\n", effect_slot_id, input_id, input_value);
+
+    tup_message_init_set_input_value_simple(&msg, effect_slot_id, input_id,
+            input_value);
+    ret = tup_context_send(&tup_ctx, &msg);
+    tup_message_clear(&msg);
+    return ret;
+}
+
+static int do_activate_internal_sensors(int argc, char *argv[])
+{
+    TupMessage msg;
+    int state_activation;
+    int ret;
+
+    if (argc != 1) {
+        printf("'activate_internal_sensors' arguments : <state active>");
+        return -EINVAL;
+    }
+
+    ret = sscanf(argv[0], "%d", &state_activation);
+    if (ret != 1) {
+        printf("failed to parse arguments\n");
+        return -EINVAL;
+    }
+    switch(state_activation) {
+        case 0:
+            printf("Internal Sensors deactivated\n");
+            break;
+        case 1:
+            printf("Internal Sensors activated\n");
```

Comment:
these prints should not be in the preterit mood, but rather in the imperative, ie:
`activate internal sensors`

as their are not activated yet.